From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Linus Groh <mail@linusgroh.de>
Date: Mon, 2 Oct 2023 18:52:48 +0100
Subject: [PATCH] Include `sys/uio.h` in `socketmodule.c`

This is to ensure that `struct iovec` is defined, which is required by
the `socket` module.
---
 Modules/socketmodule.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/socketmodule.c b/Modules/socketmodule.c
index 5e3efbd2222b3b4a8ec139df0294da9abfcf15d3..8ff1d50dfaf731c0882cfca02ab2a5c973b992d1 100644
--- a/Modules/socketmodule.c
+++ b/Modules/socketmodule.c
@@ -179,7 +179,7 @@ shutdown(how) -- shut down traffic in one or both directions\n\
 # undef HAVE_GETHOSTBYNAME_R_6_ARG
 #endif
 
-#if defined(__OpenBSD__)
+#if defined(__OpenBSD__) || defined(__serenity__)
 # include <sys/uio.h>
 #endif
 
